### ISSUE TITLE
RR-ClArC should freeze layers up to l-1 while finetuning

### ISSUE THREAD

**Issue by istepka** [FairUnlearn/debiasing#23]
Full credit for spotting this to @kasia284.

In RR-ClArC the fine-tuning procedure should freeze all layers *before* the layer on which we apply the gradient reprojection.  It's probably a minor error, because there's only one fine-tuning epoch, but still definitely worth fixing.
